src/index.html

```diff
@@ -4,9 +4,302 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    >
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="https://www.moyo.ua/" class="header__logo">
+        <img
+          src="./images/logo.png"
+          alt="moyo logo"
+          class="header__logo-img"
+        >
+      </a>
+      <nav>
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a href="link" class="header__nav-link is-active">Apple</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="link" class="header__nav-link">Samsung</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="link"
+              class="header__nav-link"
+            >Smartphones</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              href="link"
+              class="header__nav-link"
+              data-qa="nav-hover"
+            >Laptops & Computers</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="link" class="header__nav-link">Gadgets</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="link" class="header__nav-link">Tablets</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="link" class="header__nav-link">Photo</a>
+          </li>
+
+          <li class="header__nav-item">
+            <a href="link" class="header__nav-link">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main>
```

Comment:
line between siblings, please check all over the document [code style](https://mate-academy.github.io/layout_task-guideline/html-css-code-style-rules#html-formatting)